master/reference/felix/configuration.md

```diff
@@ -117,40 +117,18 @@ The Kubernetes API datastore driver reads its configuration from Kubernetes-prov
-`TyphaCN` or `TyphaURISAN` must be configured, and Felix will check the
-presented certificate accordingly.
-
--  For a [SPIFFE](https://github.com/spiffe/spiffe)-compliant deployment you
-   should configure `TyphaURISAN` with a [SPIFFE
-   Identity](https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md#2-spiffe-identity)
-   and provision Typha certificates with the same identity in their URI SAN
-   field.
-
--  Alternatively you can configure `TyphaCN` and provision Typha certificates
-   with that `TyphaCN` value in their Common Name field.
-
-If both those parameters are set, a Typha certificate only has to match one of
-them; this is intended for when a deployment is migrating from using Common
-Name to using a URI SAN, to express identity.
-
 | Configuration parameter | Environment variable   | Description | Schema |
 | ----------------------- | ---------------------- | ----------- | ------ |
-| `TyphaCAFile`           | `FELIX_TYPHACAFILE`    | The full path to the certificate file for the Certificate Authorities that Felix trusts for Felix-Typha communications. | string |
-| `TyphaCertFile`         | `FELIX_TYPHACERTFILE`  | The full path to the certificate file for this Felix instance to use to connect to Typha. | string |
-| `TyphaCN`               | `FELIX_TYPHACN`        | If set, the Common Name that Typha's certificate must have. [Default: not set] | string |
-| `TyphaKeyFile`          | `FELIX_TYPHAKEYFILE`   | The full path to the private key file for this Felix instance to use to connect to Typha. | string |
-| `TyphaURISAN`           | `FELIX_TYPHAURISAN`    | If set, a URI SAN that Typha's certificate must have. [Default: not set] | string |
+| `TyphaCAFile`           | `FELIX_TYPHACAFILE`    | Use  this parameter to supply Felix with the path to the file containing the root certificate of the CA that issued the Typha server certificate. Configures Felix to trust the signature on the certificates provided by the Typha server. Example: `/etc/felix/ca.pem` | string |
+| `TyphaCertFile`         | `FELIX_TYPHACERTFILE`  | Path to the file containing the client certificate issued to Felix. Enables Felix to participate in mutual TLS authentication and identify itself to the Typha server. Example: `/etc/felix/cert.pem` | string |
+| `TyphaCN`               | `FELIX_TYPHACN`        | If set, the `Common Name` that Typha's certificate must have. If you have enabled TLS on the communications from Felix to Typha, you must set a value here or in `TyphaURISAN`. You can set values in both, as well, such as to facilitate a migration from using one to the other. It either matches, the communication succeeds. [Default: none] | string |
+| `TyphaKeyFile`          | `FELIX_TYPHAKEYFILE`   | Path to the file containing the private key of the Felix client certificate. Enables Felix to participate in mutual TLS authentication and identify itself to the Typha server. Example: `/etc/felix/key.pem` (optional) | string |
```

Comment:
"private key of the Felix client certificate" doesn't sound right.  I think "private key matching `TyphaCertFile`" would be better.